backend/services/AccountWebhookService.ts

```diff
@@ -40,6 +41,8 @@ export function accountServiceWebhook(payload: TxWebhookRsp) {
     case TransactionType.SIWF_SIGNUP:
       requiredFields.push('handle', 'accountId');
       logger.debug(`Received account signup response for referenceId ${referenceId}`, payload as SIWFWebhookRsp);
+
+      sseManager.broadcast('account_created', payload);
```

Comment:
If we're broadcasting ALL `account_created` events, is that a security risk? Could a malicious client intercept an account creation event and use it to become logged in?